boards/arm/kite_nrf9160/kite_nrf9160_common.dts

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (c) 2023 Elektronikutvecklingsbyrån EUB AB
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "kite_nrf9160_common-pinctrl.dtsi"
+#include "kite_nrf9160-connectors-gpio.dtsi"
+#include "kite_nrf9160-connectors-pinctrl.dtsi"
+#include <dt-bindings/gpio/pmod.h>
+
+/ {
+	model = "EUB Kite-nRF9160";
+	compatible = "eub,kite-nrf9160";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio0 10 0>;
+			label = "Green LED";
+		};
+		led1: led_1 {
+			gpios = <&gpio0 9 0>;
+			label = "Orange LED";
+		};
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		pwm_led0: pwm_led_0 {
+			pwms = <&pwm0 0 PWM_MSEC(20) PWM_POLARITY_NORMAL>;
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		button0: button_0 {
+			gpios = <&gpio0 28 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "USER_BUTTON";
+		};
+	};
+
+	aliases {
+		led0 = &led0;
+		led1 = &led1;
+		green-led = &led0;
+		orange-led = &led1;
+		pwm-led0 = &pwm_led0;
+		sw0 = &button0;
+		user-button = &button0;
+		bootloader-led0 = &led0;
+		mcuboot-button0 = &button0;
+		mcuboot-led0 = &led0;
+		watchdog0 = &wdt0;
+	};
+
+	en_pmod1: pmod1-pwr-ctrl {
+		compatible = "regulator-fixed-sync", "regulator-fixed";
+		status = "okay";
+		regulator-name = "EN_PMOD1";
+
+		enable-gpios = <&gpio0 21 GPIO_ACTIVE_HIGH>;
+		regulator-boot-on;
+	};
```

Comment:
This is superseded by the code in `board.c`, no? Delete if so

Reply:
Actually the code in `board.c` is improved slightly here. Should've made that a separate patch set I guess. But now that we're here, feel free to look at `board.c` and see if it's to your liking

Reply:
Yep, I think that looks good. See separate comment.